draft-ietf-oauth-v2-1.md

```diff
@@ -487,25 +488,43 @@ Access tokens (as well as any confidential access token
 attributes) MUST be kept confidential in transit and storage, and
 only shared among the authorization server, the resource servers the
 access token is valid for, and the client to whom the access token is
-issued.  Access token credentials MUST only be transmitted using TLS
-as described in {{tls-version}} with server authentication as defined by
-{{RFC2818}}.
+issued.
 
 The authorization server MUST ensure that access tokens cannot be
 generated, modified, or guessed to produce valid access tokens by
 unauthorized parties.
 
-## TLS Version {#tls-version}
-
-Whenever Transport Layer Security (TLS) is used by this
-specification, the appropriate version (or versions) of TLS will vary
-over time, based on the widespread deployment and known security
-vulnerabilities.  Refer to {{BCP195}} for up to date recommendations on
-transport layer security.
+## Communication security {#communication-security}
+
+Implementations MUST use a mechanism to provide communication
+authentication, integrity and confidentiality such as
+Transport-Layer Security {{RFC8446}},
+to protect the exchange of clear-text credentials and tokens
+either in the payload body or in header fields
+from eavesdropping, tampering, and message forgery
+(eg. see {{client-secret}}, {{authorization_codes}} and {{token-endpoint}}).
+
+Securing the communication channel is critical
+when the authorization process is used as a form of
+delegated end-user authentication by the client (e.g., third-party
+sign-in service).
+
+OAuth URLs MUST use the `https` scheme
```

Comment:
Should the text mention that this holds true for all parties involved, including the client?